core/jobs/src/main/java/com/cognifide/aet/job/common/datafilters/jserrorsfilter/JsErrorsFilter.java

```diff
@@ -72,15 +72,24 @@ public boolean apply(@Nullable JsErrorLog input) {
   }
 
   private boolean shouldFilterOut(JsErrorLog jse) {
-    boolean sourceNotSpecified = sourceFile == null;
-    boolean sourceEquals = jse.getSourceName().equals(sourceFile);
-    boolean sourceEndsWith = jse.getSourceName().endsWith("/" + sourceFile);
-
-    return (sourceNotSpecified || sourceEquals || sourceEndsWith)
+    String source = jse.getSourceName();
+    return shouldExcludeRegardingSource(source)
         && ParamsHelper.matches(errorMessagePattern, jse.getErrorMessage())
         && ParamsHelper.equalOrNotSet(line, jse.getLineNumber());
   }
 
+  private boolean shouldExcludeRegardingSource(String errorSource) {
+    boolean shouldExclude;
+
+    boolean sourceParamNotSpecified = sourceFile == null;
+    if (sourceParamNotSpecified) {
+      shouldExclude = true;
+    } else {
+      shouldExclude = errorSource.equals(sourceFile) || errorSource.endsWith(sourceFile);
```

Comment:
`errorSource.endsWith(sourceFile);`
will return true if sourceFile is an empty string,
imo this is not expected behavior,
 maybe check if sourceFile isBlank in line 84 instead of null check.